__tests__/integration/publish.test.js

```diff
@@ -0,0 +1,93 @@
+const request = require('request-promise-native');
+
+describe('Publishes', () => {
+    const SDK_VERSION = '1.1.1';
+    const instanceId = '25c7b7c2-cabc-4e6b-b7d0-c9ad5c787e50';
+    const secretKey =
+        'A2DB402A49D7F7F45044D2049F5B2CDEB793564296B396A9B15A727ABE93EB50';
+    let baseUrl = `https://${instanceId}.pushnotifications.pusher.com/publish_api/v1/instances/${instanceId}/publishes`;
+
+    describe('Interests', () => {
+        let baseRequest;
+
+        beforeEach(() => {
+            baseRequest = request.defaults({
```

Comment:
The idea is not to create a raw request here, but rather to use the library as a customer would